scene/gui/control.cpp

```diff
@@ -2419,7 +2419,7 @@ void Control::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("_font_changed"), &Control::_font_changed);
 
 	BIND_VMETHOD(MethodInfo("_gui_input", PropertyInfo(Variant::INPUT_EVENT, "event")));
-	BIND_VMETHOD(MethodInfo(Variant::VECTOR2, "get_minimum_size"));
+	BIND_VMETHOD(MethodInfo(Variant::VECTOR2, "_get_minimum_size"));
```

Comment:
That's incorrect, you forgot to call the new method..... (you should probably edit SceneStringNames too, IDK)